lib/Jit/jitoptions.cpp

```diff
@@ -50,11 +58,52 @@ void freeStringConfigValue(ICorJitInfo *CorInfo, UTF16CharT *Value) {
 }
 
 JitOptions::JitOptions(LLILCJitContext &Context) {
-  // Set 'IsAltJit' based on environment information.
-  IsAltJit = queryIsAltJit(Context);
+
+  if (!StaticOptionsInitialized) {
```

Comment:
This block should be broken up into individual initializers for the options like the others are handled.